File: Estrutura_Condicional/C++/Ex2702/main.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    vector<int> disponiveis(3);
    vector<int> requisitadas(3);

    for (int i = 0; i < 3; ++i) {
        cin >> disponiveis[i];
    }

    for (int i = 0; i < 3; ++i) {
        cin >> requisitadas[i];
    }

    int nao_atendidos = 0;
    for (int i = 0; i < 3; ++i) {
        nao_atendidos += max(0, requisitadas[i] - disponiveis[i]);
    }

    cout << nao_atendidos << endl;

    return 0;
}

File: Estrutura_Condicional/Java/Ex2702/Main.java
package Ex2702;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] disponiveis = new int[3];
        int[] requisitadas = new int[3];

        for (int i = 0; i < 3; ++i) {
            disponiveis[i] = scanner.nextInt();
        }

        for (int i = 0; i < 3; ++i) {
            requisitadas[i] = scanner.nextInt();
        }

        int naoAtendidos = 0;

        for (int i = 0; i < 3; ++i) {
            naoAtendidos += (requisitadas[i] - disponiveis[i]) > 0 ? (requisitadas[i] - disponiveis[i]) : 0;
        }

        System.out.println(naoAtendidos);
        scanner.close();
    }
}